docs/content/getting-started/setup-openfga.mdx

```diff
@@ -188,6 +188,20 @@ If you need to serve the profiler on a different address than the default `:3001
 docker run -p 8080:8080 -p 8081:8081 -p 3000:3000 -p 3002:3002 openfga/openfga run --profiler-enabled --profiler-addr :3002
 ```
 
+## Health Check
+
+OpenFGA is configured with a HTTP and gRPC health check endpoint `/healthz` that is wired to datastore testing. Possible response values are
```

Comment:
The gRPC health "endpoint" or whatever it is called is at
```
$ grpcurl -plaintext localhost:8081 grpc.health.v1.Health/Check
{
  "status": "SERVING"
}
```
so perhaps this should say something else. Maybe like:

```suggestion
OpenFGA is configured with an HTTP health check endpoint `/healthz` and a gRPC health check `grpc.health.v1.Health/Check`, which is wired to datastore testing. Possible response values are
```

Reply:
Added suggestion and also add command line example to reach gRPC healtcheck endpoint